src/routes/events.js

```diff
@@ -8,4 +8,6 @@ router.get('/:id', eventsControllers.getEvent);
 
 router.get('/:id/related-events', eventsControllers.getRelatedEvents);
 
+router.patch('/:id', eventsControllers.updateEvent);
+
```

Comment:
only the Ngo That own the event can update it, isAuth and isNgo middlewares should be added.